app/assets/javascripts/app.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Created by cuthbertm on 3/21/16.
+ */
+toastr.options = {
```

Comment:
I did not, I am not using it very much at the moment, just one minor thing really which I will probably remove anyway. It seemed to work pretty well and easily so I didn't see a reason to change it either.